hbase-balancer/src/main/java/org/apache/hadoop/hbase/master/balancer/BalancerClusterState.java

```diff
@@ -905,6 +975,52 @@ int[] addRegion(int[] regions, int regionIndex) {
     return newRegions;
   }
 
+  int[] removeRegions(int[] regions, Set<Integer> regionIndicesToRemove) {
```

Comment:
This method, and the below addRegions, are just a nicer way to add/remove regions from the BCS arrays in bulk